templates/master/00-master/_base/files/apiserver-url-env.yaml

```diff
@@ -0,0 +1,7 @@
+mode: 0644
+path: "/etc/kubernetes/apiserver-url.env"
```

Comment:
Maybe we should add something like a "MCO internals" package that things like the SDN could vendor in that would have this as a constant.

Reply:
oh, that's a decent idea - as long as it won't annoy `go mod`.

Reply:
That said, this is certainly not changeable, so parameterizing it isn't really useful.

Up to you.